packages/theme-product/theme-product.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Find a match in the project JSON (using a ID number) and return the variant (as an Object)
+ * @param {Object} product Product JSON object
+ * @param {Number} value Accepts Number (e.g. 6908023078973)
+ * @returns {Object} The variant object once a match has been successful. Otherwise null will be return
+ */
+export function getVariantFromId(product, value) {
+  _validateProductStructure(product);
+
+  if (typeof value === 'number') {
```

Comment:
Do we want to return null if `value === 'number'` or throw an error?

Reply:
Throw an error is better to follow the same pattern used in other methods when the type is invalid. I just added `TypeError` if the value is not a number.